patches/tModLoader/Terraria/ModLoader/ModWaterStyle.cs

```diff
@@ -60,6 +60,23 @@ public abstract class ModWaterStyle : ModTexturedType
 		public virtual Color BiomeHairColor() {
 			return new Color(28, 216, 94);
 		}
+
+		/// <summary>
+		/// Returns the texture to be used when drawing rain of this water type.
+		/// Default uses the vanilla rain texture.
+		/// </summary>
```

Comment:
Line breaks in xml summaries should be prepended with `<br/>`: `<br/> Default uses the vanilla rain texture.`